src/main/java/org/terasology/scenario/components/TextComponent.java

```diff
@@ -13,12 +13,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.terasology.scenario.components.actions;
+package org.terasology.scenario.components;
 
 import org.terasology.entitySystem.Component;
 import org.terasology.network.FieldReplicateType;
 import org.terasology.network.Replicate;
 
+/**
+ * Component that includes the text that is parsed for arguments

```

Comment:
A link to a palce where there is more information or more information here would be good. e.g. what format must arguments have to be recognized as such and how to find out what types areguments can have